src/styles/utils/_media-queries.scss

```diff
@@ -0,0 +1,23 @@
+@media screen and (max-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(1, $card-width);
+  }
+}
+
+@media screen and (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, $card-width);
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .catalog {
+    grid-template-columns: repeat(3, $card-width);
+  }
+}
+
+@media screen and (min-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(4, $card-width);
+  }
+}
```

Comment:
I think there are no need to create a separate file for media queries,  we can put them under general catalog class using scss nesting